File: tools-manage/invocationlab-admin/src/main/java/org/javamaster/invocationlab/admin/model/erd/ModulesBean.java
package org.javamaster.invocationlab.admin.model.erd;

import lombok.Data;
import lombok.NoArgsConstructor;

import java.util.List;

/**
 * @author yudong
 * @date 2023/2/12
 */
@NoArgsConstructor
@Data
public class ModulesBean {
    private String name;
    private String chnname;
    private Integer sort;
    private List<EntitiesBean> entities;
    private GraphCanvasBean graphCanvas;
    private List<AssociationsBean> associations;
}

File: tools-manage/invocationlab-admin-21/src/main/kotlin/org/javamaster/invocationlab/admin/model/erd/ModulesBean.kt
package org.javamaster.invocationlab.admin.model.erd

import org.javamaster.invocationlab.admin.annos.AllOpen

/**
 * @author yudong
 * @date 2023/2/12
 */


@AllOpen
class ModulesBean {
    var name: String? = null
    var chnname: String? = null
    var sort: Int? = null
    var entities: List<EntitiesBean>? = null
    var graphCanvas: GraphCanvasBean? = null
    var associations: List<AssociationsBean>? = null
}
